src/jira.py

```diff
@@ -137,21 +152,40 @@ def update_remote(self):
     def post_new_issue(self):
         """Post this issue to Jira for the first time. The issue must not already exist."""
 
-        r = requests.post(self.url + "issue/", headers=self.headers, json=self.dict_format())
+        r = requests.post(f"{self.url}issue/", headers=self.headers, json=self.dict_format())
 
         if r.status_code != 201:  # HTTP 201 means created
             print(f"{r.status_code} Error")
 
         self.jira_key = r.json()["key"]  # keep the key that Jira assigned to this issue when creating it
 
-    def add_to_epic(self, epic_key):
-        """Make this issue part of a Jira epic. If it is already in an epic, that will be overwritten."""
-        issues = {'issues': [self.jira_key]}
-
+    def add_to_this_epic(self, issue_key):
+        """Make the given issue belong to this epic (self). If it is already in an epic, that will be overwritten."""
+        issues = {'issues': [issue_key]}
+        old_api_url = self.url.split('api')[0]  # remove 'api/latest' from the url
```

Comment:
Use `first(...` from `from import more_itertools import first` instead of `[0]`. Easier to read.

Reply:
done